src/topojson/topojson_intf.ml

```diff
@@ -1 +1,230 @@
+
+    val lat : t -> float
+    (** The latitude value of the position *)
+
+    val altitude : t -> float option
+    (** Optional altitude/elevation value of the position *)
+
+    val equal : t -> t -> bool
+    (** Whether two positions are equal by comparing each value *)
+
+    val v : ?altitude:float -> lng:float -> lat:float -> unit -> t
+    (** A position constructor *)
+  end
+
+  module Arcs : sig
+    type t
+    (** Arcs is an array of arc indexes *)
+  end
+
+  module Point : sig
+    type t
+    (** A point is a single {!Position.t} *)
+
+    val position : t -> Position.t
+    (** Convert a point to a position *)
+
+    val v : Position.t -> t
+    (** Create a poitn from a position. *)
+  end
+
+  module MultiPoint : sig
+    type t
+    (** A multipoint is an array of positions. *)
+
+    val coordinates : t -> Position.t array
+    (** Get the positions that make up this multipoint object. *)
+
+    val v : Position.t array -> t
+    (** Create a multipoint object from an array of positions. *)
+  end
+
+  module LineString : sig
+    type t
+    (** A line string is two or more points *)
+
+    val v : Arcs.t -> t
+    (** Creates a line string from arc indexes *)
+  end
+
+  module MultiLineString : sig
+    type t
+    (** A collection of line strings *)
+
+    val v : LineString.t array -> t
+    (** Create a multiline string *)
+  end
+
+  module Polygon : sig
+    type t
+    (** A close loop with optional rings *)
+
+    val rings : t -> LineString.t array
+    (** [rings t] returns the linear rings contained in [t] (a Polygon object) *)
+
+    val exterior_ring : t -> LineString.t
+    (** [exterior_ring t] returns the first linear ring contained in [t] (a
+        Polygon object). This ring bounds the surface *)
+
+    val interior_rings : t -> LineString.t array
+    (** If [t] (a Polygon object) contains more than 1 linear ring,
+        [interior_rings t] returns the rest of the linear rings apart from the
+        first. These rings (if present), bound the holes. *)
+
+    val v : LineString.t array -> t
+    (** Create a polygon object from an array of close line strings *)
+  end
+
+  module MultiPolygon : sig
+    type t
+    (** A multi-polygon object *)
+
+    val polygons : t -> Polygon.t array
+    (** Access the polygons *)
+
+    val v : Polygon.t array -> t
+    (** Create a multi-polygon object from an array of {!Polygon.t}s *)
+  end
+
+  type geometry =
+    | Point of Point.t
+    | MultiPoint of MultiPoint.t
+    | LineString of LineString.t
+    | MultiLineString of MultiLineString.t
+    | Polygon of Polygon.t
+    | MultiPolygon of MultiPolygon.t
+    | Collection of t list
+
+  and t = geometry * (string * json) list
+
+  val foreign_members : t -> (string * json) list
+  (** [foreign_members t] will extract name/value pair of a foreign member from
+      t (a topojson object) *)
+
+  include Json_conv with type t := t and type json := json
+end
+
+module type Topology = sig
```

Comment:
Do we still need this, `Topology` is defined in `S`

Reply:
Yeah sure, we don't need this, just forgot to remove ...